File: RepoTransBench/target_projects/Java/C#/sockeqwe_fragmentargs/tests/original/FragmentArgsInjectorTest.cs
using Xunit;

namespace ProjectName.Tests.Original
{
    public class FragmentArgsInjectorTest
    {
        public interface IFragmentArgsInjector
        {
            void Inject(object target);
        }

        [Fact]
        public void InjectorInterfaceShouldAllowAnyObject()
        {
            IFragmentArgsInjector inj = new DummyInjector();
            inj.Inject("dummy");
        }

        private class DummyInjector : IFragmentArgsInjector
        {
            public void Inject(object target)
            {
                // no-op
            }
        }
    }
}

File: RepoTransBench/target_projects/Java/Go/sockeqwe_fragmentargs/tests/original/fragmentargs_injector_test.go
package original

import (
	"testing"
)

func TestInjectorInterfaceShouldAllowAnyObject(t *testing.T) {
	inj := &anyObjectInjector{}
	inj.Inject("dummy")
}

type anyObjectInjector struct{}

func (a *anyObjectInjector) Inject(target interface{}) {
	// no-op, just check interface accepts any object
}